knees4bees/rancid-tomatillos
# Feature/landing page styles

## Is this a fix, feature, or refactor?
This is a feature

## What is the change?
Styles have been added to the landing page

## What does it fix?
n/a

## Where should the reviewer start?
Read over all component 

## How should this be tested?
Open up the web app make sure everything looks ok and is responsive.

More responsive design will be added further down the line.

## Relevant ScreenShots
n/a
